analyses/bangladesh/trigger_development/bgd-trigger-part2.md

```diff
@@ -0,0 +1,363 @@
+```python
+from importlib import reload
+from datetime import datetime, timedelta
+
+import matplotlib.pyplot as plt
+import matplotlib as mpl
+import pandas as pd
+import numpy as np
+from scipy.stats import norm, pearsonr
+
+
+import read_in_data as rd
+reload(rd)
+
+mpl.rcParams['figure.dpi'] = 300
+```
+
+### Create GloFAS objects
+
+```python
+da_glofas_reanalysis = rd.get_glofas_reanalysis()
```

Comment:
This is probably because I'm just a couple days behind in reviewing this but just getting an error here as this isn't updated for the new v2/v3 file naming.

Reply:
Oops sorry about that! Just needed to merge in develop, should work now.